src/ChunkingContext.jl

```diff
@@ -0,0 +1,82 @@
+# When splitting the work among multiple tasks, each task should have at least this many bytes of input
+# This is to avoid having too many tasks with too little work to do.
+# TODO: make this configurable and find a good default (the current 16 KiB is a guess)
+const MIN_TASK_SIZE_IN_BYTES = 16 * 1024
+
+_comment_to_bytes(x::AbstractString) = Vector{UInt8}(x)
```

Comment:
should this be `codeunits(x)` instead of `Vector{UInt8}(x)`?

Reply:
😅 sorry it's hard not to do code review while i'm here - if you do want to change it, definitely ignore this comment and do it in a separate PR. Sorry! 😁

Reply:
Since the `comment` field of `ChunkingContext` is `Vector{UInt8}`, there is no benefit to using `codeunits`.